### ISSUE TITLE
Using DWD with the Admin SDK on GCE

### ISSUE THREAD

**Issue by mirons-google** [google/oauth2client#707]
I'm struggling to find the proper way to create credentials that can be used with the Admin SDK (directory api).

TL;DR: How can I use DWD on GCE using default credentials?

Using this API required enabling domain-wide delegation (DWD) and that's been done. The code [here](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/google/cloud/security/inventory/pipelines/load_groups_pipeline.py#L86) works well for local installations of Forseti (with the properly supplied flags).

However, I'm unable to find the proper oauth flow for doing this on GCE. I've properly given the GCE instance the correct scope `https://www.googleapis.com/auth/admin.directory.group.readonly`. I'd assume I could use the GCE credentials classes, but there seems to be no class that also allows me to call .make_delegated() which is required for the API.

**Comment by jonparrott** [comment]
GCE credentials can not sign bytes which is needed in order to generate the assertion necessary to do domain-wide delegation. You'll need to use a service account keyfile.
